packages/peregrine/lib/talons/Checkout/useFlow.js

```diff
@@ -31,18 +33,25 @@ export const useFlow = props => {
         {
             beginCheckout,
             cancelCheckout,
+            setCountries,
             submitOrder,
             submitPaymentMethodAndBillingAddress,
             submitShippingAddress,
             submitShippingMethod
         }
     ] = useCheckoutContext();
-    const { onSubmitError, setStep } = props;
+    const { countriesQuery, onSubmitError, setStep } = props;
+
+    const [runQuery, queryResponse] = useLazyQuery(countriesQuery);
+    const getCountries = runQuery;
+    const { data: countriesData } = queryResponse;
 
     const handleBeginCheckout = useCallback(async () => {
+        getCountries();
```

Comment:
I think we can just do this as a query on mount rather than wait for checkout to begin.

Reply:
Done: https://github.com/magento/pwa-studio/pull/1993/commits/68cd2b8ac59650c6a8b8a677a46d355adf1d1c0b.